src/ClusterProviders/Proto.Cluster.Consul/ConsulProvider.cs

```diff
@@ -6,6 +6,7 @@
 

```

Comment:
I think "StartConsulDevMode should be moved into the cluster demo project instead of having this in the provider library. I guess this will only be used when starting the demo? @rogeralsing